gamedb.yaml

```diff
@@ -554,7 +556,9 @@
 - name: Control
   platform: epic-store
   id: Calluna
-  banner: https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/05de265b83af799ad4f6753aff62d1ee.png
+  banner: steam:870780
```

Comment:
This is a different game. "Control: Ultimate Edition" vs "Control". Steam does not have "Control".

Reply:
Oh sorry.  I mixed these versions.